<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1316</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="plotPointCloud" native="true">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>800</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plotPeopleCount" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>800</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>point cloud</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>people</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStartDetect">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>开始检测</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRadarOff">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSendConfig">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>加载配置</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string> 红：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdRed">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>96</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>蓝：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdBlue">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>96</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>人员计数：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdTotal">
       <property name="minimumSize">
        <size>
         <width>132</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="digitCount">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1316</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionAboutUs"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="actionRadarConfig"/>
    <addaction name="actionSerialSetting"/>
    <addaction name="actionAreaSetting"/>
    <addaction name="checkRedBlueBox"/>
    <addaction name="checkFlipUpDown"/>
    <addaction name="checkFlipLeftRight"/>
    <addaction name="actionDefaultSetting"/>
   </widget>
   <addaction name="menu_3"/>
   <addaction name="menu"/>
  </widget>
  <action name="actionSerialSetting">
   <property name="text">
    <string>串口设置</string>
   </property>
  </action>
  <action name="actionAboutUs">
   <property name="text">
    <string>关于我们</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>关于Qt</string>
   </property>
  </action>
  <action name="actionAreaSetting">
   <property name="text">
    <string>视场设置</string>
   </property>
  </action>
  <action name="checkRedBlueBox">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>红蓝框</string>
   </property>
  </action>
  <action name="actionRadarConfig">
   <property name="text">
    <string>选择其他配置文件</string>
   </property>
  </action>
  <action name="checkFlipUpDown">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>上下翻转</string>
   </property>
  </action>
  <action name="checkFlipLeftRight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>左右翻转</string>
   </property>
  </action>
  <action name="actionDefaultSetting">
   <property name="text">
    <string>恢复默认场景</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
